README.md

```diff
@@ -14,6 +14,26 @@ library and the functionality it provides.
 
 As of version `v5.0.0`, `joda-time` has been deprecated in favor of java-time so you might see a different format for the string output of date parameters and some functions might not be available anymore. For more reference view this PR([#173](https://github.com/omise/omise-java/pull/173))
 
+# Versioning and Compatibility
+
+Although you can use omise-java with other Java versions, we have validated the following versions in our CI pipeline to ensure compatibility and stability.
+
+### Java Runtime Version(s) Supported
+
+Represents the java version that needs to be installed on your local in order to be able to run the precompiled java byte code inside your project. i.e Installing the SDK in your project from maven and running it in your custom project.
+
+### Minimum Java Version Required for Compilation
+
+Represents the Java version required to compile the SDK after you clone the SDK's repo or fork it.
+
+| Library Version | Support           | Supported Omise API Version | Java Runtime Version(s) Supported | Minimum Java Version Required for Compilation | Compilation Tested On | Notes                                                                                  |
```

Comment:
Can we shorten some headers? How about we do the following changes?

- Java Runtime Version(s) Supported => Java Runtime OR JRE
  - Use `>`, `>=`, `<`, and `<=` to indicate the min or max version supported
- Minimum Java Version Required for Compilation => Java Version (Compilation)
  - Use `>`, `>=`, `<`, and `<=` to indicate the min or max version supported
- Compilation Tested On => Compilation Tested